CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed an issue when selecting a step from the steps list, you could be brought to the wrong step. [#1524](https://github.com/mapbox/mapbox-navigation-ios/pull/1524/)
 * `StyleManager.locationFor(styleManager:)` now allows for an optional CLLocation to be returned. [#1523](https://github.com/mapbox/mapbox-navigation-ios/pull/1523)
 * NavigationViewController now uses the recommended way `.preferredStatusBarStyle` to set the style of the status bar. [#1535](https://github.com/mapbox/mapbox-navigation-ios/pull/1535)
+* `InstructionsBannerView.updateDistance(for:)` currently updates the instruction banner's distance information on every location update. Also, `InstructionsBannerView.set(_:)` has been optimized to only modify the displayed instruction details when the `visualInstructionIndex` for the current route step is updated by the `RouteController.updateVisualInstructionProgress()`. 
```

Comment:
Let's make this more clear to developers:

* We removed `InstructionsBannerView.update(_:)` which previously updated the banner instruction as well as the distance.
* This has been been replaced with `InstructionsBannerView.updateDistance(for:)` and also `insert_new_func_name`. `InstructionsBannerView.updateDistance(for:)` should be called on every location update, while `insert_new_func_name` should be called when `NSNotification.Name. routeControllerDidPassVisualInstructionPoint` is emitted. 
* No need to talk about visualInstructionIndex or `RouteController.updateVisualInstructionProgress()`